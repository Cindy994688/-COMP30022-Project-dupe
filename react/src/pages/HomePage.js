import React, {Component} from 'react'
//import {View} from 'react-native'
import Jumbotron from 'react-bootstrap/Jumbotron'
import './darkmode.css'
import './pages.css'
import './access.css'

import Contact from './Contact.js'
import ToggleProj from './ToggleProj.js'
import ToggleLeft from './ToggleLeft.js'
import ToggleRight from './ToggleRight.js'
import AliceCarousel from 'react-alice-carousel'
import 'react-alice-carousel/lib/alice-carousel.css'
import ChaoHover from './chaohover.png'
import RonHover from './ronhover.png'
import MusHover from './mushover.png'
import XuHover from './xuhover.png'
import MengHover from './menghover.png'
import { Image } from 'react-native';
import { Link, Element } from 'react-scroll';


//import axios from 'axios'
//import {clientBanner} from './clientbanner.jpg'
//const clientBanner = "../../src/pages/clientbanner.jpg";


class HomePage extends Component {
  constructor() {
    super();
    this.state = {
      galleryItems: [
      <img src="/image/ChaoWei.jpg" alt="Image of Chao-Wei Chiang on an image carousel"
        onMouseOver={e => (e.currentTarget.src = ChaoHover)}
        onMouseLeave={e => (e.currentTarget.src = "/image/ChaoWei.jpg")}/>,
      <img src="/image/Mengyan.jpg" alt="Image of Meng-Yan Hou on an image carousel"
        onMouseOver={e => (e.currentTarget.src = MengHover)}
        onMouseLeave={e => (e.currentTarget.src = "/image/Mengyan.jpg")}/>,
      <img src="/image/MustafaFullBody.jpg" alt="Image of Mustafa Awni on an image carousel"
        onMouseOver={e => (e.currentTarget.src = MusHover)}
        onMouseLeave={e => (e.currentTarget.src = "/image/MustafaFullBody.jpg")}/>,
      <img src="/image/Ron.jpg" alt="Image of Ron Chiu on an image carousel"
        onMouseOver={e => (e.currentTarget.src = RonHover)}
        onMouseLeave={e => (e.currentTarget.src = "/image/Ron.jpg")}/>,
      <img src="/image/Xu.jpg" alt="Image of Xu Han on an image carousel"
        onMouseOver={e => (e.currentTarget.src = XuHover)}
        onMouseLeave={e => (e.currentTarget.src = "/image/Xu.jpg")}/>
       ].map((i) => <div key={i}>{i}</div>),
       colourMode: "",
    }
  }

    // Keep the below definition, and always remember to provide a return to default option in the colour buttons
  // Make a new function for each colourscheme like the one below
  //
  //  setDarkMode= () => {
  //    this.setState({colourMode: 'Dark'});
  //  };
  //
  // And don't forget to add a button like below referencing your new switch function
  //
  // <button onClick={this.setDarkMode}>Set Dark Mode</button>
  //
  // And remember your default button!
  //
  // <button onClick={this.setDefaultMode}>Set default Mode</button>


  setColourMode = (event) => {
    const { value } = event.target;
    this.setState({colourMode: value});
  };

  handleNavClick = (event) => {
    if(this.contactReference.current){
      this.contactReference.current.scrollIntoView({
        behaviour: "smooth",
        block: "nearest"
      });
    }
  }

  toLogin = () => {
    this.props.history.push('/Login');
  }

  responsive = {
    0: { items: 1 },
    1024: { items: 3 },
  }

  stagePadding =   {
    paddingLeft: 500,     // in pixels
    paddingRight: 25
  }

  style = {
    display: 'flex',
    justifyContent: 'space-between',
    zIndex: 10,
    distanceFromBottom: 0,
    marginLeft:1000,
    fontFamily: 'Arial',
    fontSize: 25,
    align: 'right',
    position: 'fixed',
    color: '#8B008B'
  }


render () {
    return (
      <div className = {"fullPage" + this.state.colourMode}>
        <Element name="top">
          <Jumbotron className = {"banner" + this.state.colourMode}>
            <div className = {"maintitle" + this.state.colourMode}>
            <h1>SWEN90016 </h1>
            <div id="subheading">Group CRXMM</div>
            </div>
            {/*<p> really awesome people </p>*/}
          </Jumbotron>
        </Element>

      <div className = "buttonBox">
        <button className = {"modeButton" + this.state.colourMode} value="" onClick={this.setColourMode}>Set Default Mode</button>
        <button className = {"modeButton" + this.state.colourMode} value="AC" onClick={this.setColourMode}>Set Accessibility Mode</button>
        <button className = {"modeButton" + this.state.colourMode} value="Dark" onClick={this.setColourMode}>Set Dark Mode</button>
        <button className = {"modeButton" + this.state.colourMode} onClick={this.toLogin}>Login</button>
      </div>

      {/*NAVBAR HERE*/}
      <div className = "buttonBox">
        <Link activeClass="active" className={"modeButton" + this.state.colourMode} to="top" spy={true} smooth={true} duration={500} offset={0}>Top</Link>
        <Link activeClass="active" className={"modeButton" + this.state.colourMode} to="intro" spy={true} smooth={true} duration={500} offset={270}>Intro</Link>
        <Link activeClass="active" className={"modeButton" + this.state.colourMode} to="project" spy={true} smooth={true} duration={500} offset={450}>Project</Link>
        <Link activeClass="active" className={"modeButton" + this.state.colourMode} to="ChaoWei" spy={true} smooth={true} duration={500} offset={-25}>Harry</Link>
        <Link activeClass="active" className={"modeButton" + this.state.colourMode} to="Ron" spy={true} smooth={true} duration={500} offset={50}>Ron</Link>
        <Link activeClass="active" className={"modeButton" + this.state.colourMode} to="Mustafa" spy={true} smooth={true} duration={500} offset={-25}>Mustafa</Link>
        <Link activeClass="active" className={"modeButton" + this.state.colourMode} to="Xu" spy={true} smooth={true} duration={500} offset={50}>Zoe</Link>
        <Link activeClass="active" className={"modeButton" + this.state.colourMode} to="Mengyan" spy={true} smooth={true} duration={500} offset={-25}>Kelly</Link>
        <Link activeClass="active" className={"modeButton" + this.state.colourMode} to="contactUs" spy={true} smooth={true} duration={500} offset={0}>Contact Us</Link>
      </div>

      <p>&nbsp;&nbsp;</p>
        <div className = {"alice" + this.state.colourMode}>
        <AliceCarousel
        items={this.state.galleryItems}
        responsive={this.responsive}
        controlsStrategy="responsive"
        autoPlayInterval={2000}
        autoPlayDirection="ltr"
        autoPlay={true}
        fadeOutAnimation={true}
        /*mouseTrackingEnabled={true}*/
        playButtonEnabled={true}
        /*disableAutoPlayOnAction={true}*/
        stopAutoPlayOnHover={false}
        onSlideChange={this.onSlideChange}
        onSlideChanged={this.onSlideChanged}
        buttonsDisabled={true}
        />
        </div>

        <div className = {"divider" + this.state.colourMode}>
        <div className= {"dividerTitleSmall" + this.state.colourMode}>
        <h1> - Harry - Ron - Mustafa - Kelly - Zoe -</h1>
        </div>
        </div>

        <div className={"punk" + this.state.colourMode}>
          <Image  source="/image/logo.jpg" style={{width: 291.5, height: 107.9}}/>
        </div>

        <Element name="intro">
          <div className={"intro" + this.state.colourMode}>
          <div className={"title" + this.state.colourMode} id="intro">Intro</div>
          <p>Hi, this is CRXMM, we are a team of
             5 from the University of Melbourne.
            While we have different backgrounds,
            different passions, and are even in
            different timezones, we are in unity.
            We believe that as individuals we are
            strong, but together we are incredible.
          </p>
          </div>
        </Element>

        {/*<ToggleB>hello</ToggleB>*/}
        <Element name="project">
          <ToggleProj colourMode = {this.state.colourMode}>
          <div className = {"drop-down" + this.state.colourMode + " intro" + this.state.colourMode}>
            In this project, we help Beth, a local business owner of beauty care services
            in Melbourne, build the e-commerce website for her business. It is expected
            that the system can not just make the appointment procedure automation but also
            help promote the business to different regions.
            <div id="drop-down introtext">
            Team roles: Project Manager - Kelly; Risk Manager - Hsin-Jung;
              Test Manager - Mustafa;
              Developer - Chao-Wei and Zoe
            </div>
          </div>
        </ToggleProj>
      </Element>



      <div className={"title" + this.state.colourMode} id="whoweare">Who we are</div>
        {/*first person*/}
        <Element name="ChaoWei">
          <ToggleLeft colourMode = {this.state.colourMode} name = "ChaoWei" author = "chaowei"/>
        </Element>

        {/*second person*/}
        <Element name="Ron">
          <ToggleRight colourMode = {this.state.colourMode} name = "Ron" author = "ronchiu"/>
        </Element>

        {/*third person*/}
        <Element name="Mustafa">
          <ToggleLeft colourMode = {this.state.colourMode} name = "Mustafa" author = "mustafa"/>
        </Element>

        {/*fourth person*/}
        <Element name="Xu">
          <ToggleRight colourMode = {this.state.colourMode} name = "Xu" author = "xuhan"/>
        </Element>

        {/*fifth person*/}
        <Element name="Mengyan">
          <ToggleLeft colourMode = {this.state.colourMode} name = "Mengyan" author = "mengyan"/>
        </Element>

        <div className={"getInTouchHeader" + this.state.colourMode}>Contact Us</div>

        <Element name="contactUs">
        <Contact colourMode = {this.state.colourMode} ref={this.contactReference}/>
        </Element>


        <div className= {"footer" + this.state.colourMode}><p>last edited: 27/10/2020</p></div>


      </div>

    )
  }
}

export default HomePage;
