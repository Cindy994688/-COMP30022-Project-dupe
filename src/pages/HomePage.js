import React, {Component} from 'react'
//import {View} from 'react-native'
import { Link } from 'react-router-dom'
import Jumbotron from 'react-bootstrap/Jumbotron'
import Carousel from 'react-elastic-carousel'
import './pages.css'
import Profile from './Profile.js'
import Biotext from './Biotext.js'
import Card from '../../src/components/Card.js'

class HomePage extends Component {
  state = {
    items: [
      {id: 1, title: 'item #1'},
      {id: 2, title: 'item #2'},
      {id: 3, title: 'item #3'},
      {id: 4, title: 'item #4'},
      {id: 5, title: 'item #5'}
    ]
  }
render () {
  const { items } = this.state;
    return (
      <div>
        <Jumbotron className = "banner">
        <div className="title"><h1>EPORTFOLIO WOOT WOOT</h1></div>
        <p> really inspiring tagline </p>
        </Jumbotron>

        <div>
          <p>&nbsp;&nbsp;&nbsp;&nbsp;</p>
       
        <Carousel className = "space">
          <Card />
          <Card />
          <Card />
        </Carousel>
        <p>&nbsp;&nbsp;&nbsp;&nbsp;</p>
        </div>

        <div className="intro">
        <p> this is just a block of text. lorem ipsum i guess. or I can just write this stuff myself.
          The french word for 'bird' is "oiseau". it kind of horrifies me that someone managed to fit
          ALL THE VOWELS in such a short amount of space. wtf. truly frightening. There's also "oeuf" 
          which is egg i think. Less horrifying but still pretty bad. Wow. Also i love names like
          "Saoirse" and "Annaleise" bc that vowel ratio. Truly superb. The enemy of my enemy is not 
          my friend. They're literally just the enemy of my enemy. Just like no news != good news.
          no news == no news. end of story. 
        </p>
        </div>


        <div className="container">
          <p>
          <Link to="/users">users</Link> is a link i guess \o/
          </p>
        </div>
        {/*first person*/}
        <div style={{display: 'inline-block'}}>
          <Profile />
          </div>
        <div style={{display: 'inline-block'}}>
          <Biotext />
          </div>

          {/*second person*/}
          <div style={{display: 'inline-block'}}>
          <Biotext />
          </div>
        <div style={{display: 'inline-block'}}>
          
          <Profile />
          </div>
        
      </div>
    )
  }
}

export default HomePage;