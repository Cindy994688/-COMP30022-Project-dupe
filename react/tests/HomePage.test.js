/**
 * @jest-environment jsdom
 */

import React from 'react';
import renderer from 'react-test-renderer';
import HomePage from '../src/pages/HomePage';

test('it renders correctly', () => {
    component = renderer.create(<HomePage/>)
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
});



