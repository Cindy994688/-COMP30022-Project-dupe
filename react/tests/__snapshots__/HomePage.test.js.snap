// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it renders correctly 1`] = `
<div
  className="fullPage"
>
  <div
    className="banner jumbotron"
  >
    <div
      className="maintitle"
    >
      <h1>
        SWEN90016 
      </h1>
      <div
        id="subheading"
      >
        Group CRXMM
      </div>
    </div>
  </div>
  <div
    className="buttonBox"
  >
    <button
      className="modeButton"
      onClick={[Function]}
    >
      Set Default Mode
    </button>
    <button
      className="modeButton"
      onClick={[Function]}
    >
      Set Accessibility Mode
    </button>
    <button
      className="modeButton"
      onClick={[Function]}
    >
      Set Dark Mode
    </button>
  </div>
  <p>
      
  </p>
  <div
    className="alice"
  >
    <div
      className="alice-carousel"
    >
      <div>
        <div
          className="alice-carousel__wrapper"
          onMouseEnter={[Function]}
          onMouseLeave={[Function]}
          style={
            Object {
              "height": false,
              "paddingLeft": "0px",
              "paddingRight": "0px",
              "transition": false,
            }
          }
        >
          <ul
            className="alice-carousel__stage"
            style={
              Object {
                "height": undefined,
                "transform": "translate3d(0px, 0, 0)",
                "transition": "transform 0ms ",
              }
            }
          >
            <li
              className="alice-carousel__stage-item"
              style={
                Object {
                  "width": "0px",
                }
              }
            >
              <div>
                <img
                  alt="Image of Xu Han on an image carousel"
                  onMouseLeave={[Function]}
                  onMouseOver={[Function]}
                  src="/image/Xu.jpg"
                />
              </div>
            </li>
            <li
              className="alice-carousel__stage-item __active"
              style={
                Object {
                  "width": "0px",
                }
              }
            >
              <div>
                <img
                  alt="Image of Chao-Wei Chiang on an image carousel"
                  onMouseLeave={[Function]}
                  onMouseOver={[Function]}
                  src="/image/ChaoWei.jpg"
                />
              </div>
            </li>
            <li
              className="alice-carousel__stage-item"
              style={
                Object {
                  "width": "0px",
                }
              }
            >
              <div>
                <img
                  alt="Image of Meng-Yan Hou on an image carousel"
                  onMouseLeave={[Function]}
                  onMouseOver={[Function]}
                  src="/image/Mengyan.jpg"
                />
              </div>
            </li>
            <li
              className="alice-carousel__stage-item"
              style={
                Object {
                  "width": "0px",
                }
              }
            >
              <div>
                <img
                  alt="Image of Mustafa Awni on an image carousel"
                  onMouseLeave={[Function]}
                  onMouseOver={[Function]}
                  src="/image/MustafaFullBody.jpg"
                />
              </div>
            </li>
            <li
              className="alice-carousel__stage-item"
              style={
                Object {
                  "width": "0px",
                }
              }
            >
              <div>
                <img
                  alt="Image of Ron Chiu on an image carousel"
                  onMouseLeave={[Function]}
                  onMouseOver={[Function]}
                  src="/image/Ron.jpg"
                />
              </div>
            </li>
            <li
              className="alice-carousel__stage-item"
              style={
                Object {
                  "width": "0px",
                }
              }
            >
              <div>
                <img
                  alt="Image of Xu Han on an image carousel"
                  onMouseLeave={[Function]}
                  onMouseOver={[Function]}
                  src="/image/Xu.jpg"
                />
              </div>
            </li>
            <li
              className="alice-carousel__stage-item"
              style={
                Object {
                  "width": "0px",
                }
              }
            >
              <div>
                <img
                  alt="Image of Chao-Wei Chiang on an image carousel"
                  onMouseLeave={[Function]}
                  onMouseOver={[Function]}
                  src="/image/ChaoWei.jpg"
                />
              </div>
            </li>
          </ul>
        </div>
      </div>
      <ul
        className="alice-carousel__dots"
      >
        <li
          className="alice-carousel__dots-item __active"
          onClick={[Function]}
        />
        <li
          className="alice-carousel__dots-item"
          onClick={[Function]}
        />
        <li
          className="alice-carousel__dots-item"
          onClick={[Function]}
        />
        <li
          className="alice-carousel__dots-item"
          onClick={[Function]}
        />
        <li
          className="alice-carousel__dots-item"
          onClick={[Function]}
        />
      </ul>
      <div
        className="alice-carousel__play-btn"
      >
        <div
          className="alice-carousel__play-btn-wrapper"
        >
          <div
            className="alice-carousel__play-btn-item __pause"
            onClick={[Function]}
          />
        </div>
      </div>
    </div>
  </div>
  <div
    className="divider"
  >
    <div
      className="dividerTitleSmall"
    >
      <h1>
         - Harry - Ron - Mustafa - Kelly - Zoe -
      </h1>
    </div>
  </div>
  <div
    className="punk"
  >
    <Image
      source="/image/logo.jpg"
      style={
        Object {
          "height": 107.9,
          "width": 291.5,
        }
      }
    />
  </div>
  <div
    className="intro"
  >
    <div
      className="title"
      id="intro"
    >
      Intro
    </div>
    <p>
      Hi, this is CRXMM, we are a team of 5 from the University of Melbourne. While we have different backgrounds, different passions, and are even in different timezones, we are in unity. We believe that as individuals we are strong, but together we are incredible.
    </p>
  </div>
  <div>
    <div
      className="topTriangle thirdColor"
    >
      <div
        className="intro title"
      >
         Our Project 
      </div>
      <div
        className="sub"
      >
         Beth’s Beauty - Your best beauty services 
      </div>
      <div
        className="buttondiv"
      >
        <button
          className="btn btn-primary button"
          onClick={[Function]}
          style={
            Object {
              "cursor": "pointer",
            }
          }
          type="submit"
        >
          learn about our project
        </button>
      </div>
      <div>
          
      </div>
    </div>
  </div>
  <div
    className="title"
    id="whoweare"
  >
    Who we are
  </div>
  <div
    className="topTriangle container"
  >
    <div
      className="row"
    >
      <div
        className="col-md-6"
      >
        <div
          className="profileImage"
        >
          <Image
            source="/image/ChaoWei.jpg"
            style={
              Object {
                "borderRadius": 150,
                "height": 300,
                "width": 300,
              }
            }
          />
          <div
            className="profileName"
          >
            
          </div>
        </div>
      </div>
      <div
        className="col-md-6"
      >
        <div
          className="bioText"
        >
          
        </div>
        <div
          className="bioText"
        >
          
        </div>
        <div
          className="bioText"
        >
          
        </div>
      </div>
      <div
        className="row"
      >
        <div
          className="buttondiv"
        >
          <button
            className="btn btn-primary button"
            onClick={[Function]}
            style={
              Object {
                "cursor": "pointer",
              }
            }
            testID="leftCVbutton"
            type="submit"
          >
            See CV
          </button>
          <button
            className="btn btn-primary button"
            onClick={[Function]}
            style={
              Object {
                "cursor": "pointer",
              }
            }
            testID="leftProjectbutton"
            type="submit"
          >
            See Personal Projects
          </button>
        </div>
      </div>
    </div>
  </div>
  <div
    className="bottomTriangle bgSecondary container"
  >
    <div
      className="row"
    >
      <div
        className="col-md-6"
      >
        <div
          className="profileImage"
        >
          <Image
            source="/image/Ron.jpg"
            style={
              Object {
                "borderRadius": 150,
                "height": 300,
                "width": 300,
              }
            }
          />
          <div
            className="profileName"
          >
            
          </div>
        </div>
      </div>
      <div
        className="col-md-6"
      >
        <div
          className="bioText"
        >
          
        </div>
        <div
          className="bioText"
        >
          
        </div>
        <div
          className="bioText"
        >
          
        </div>
      </div>
      <div
        className="row"
      >
        <div
          className="buttondiv"
        >
          <button
            className="btn btn-primary button"
            onClick={[Function]}
            style={
              Object {
                "cursor": "pointer",
              }
            }
            testID="rightCVbutton"
            type="submit"
          >
            See CV
          </button>
          <button
            className="btn btn-primary button"
            onClick={[Function]}
            style={
              Object {
                "cursor": "pointer",
              }
            }
            testID="rightProjectbutton"
            type="submit"
          >
            See Personal Projects
          </button>
        </div>
      </div>
    </div>
  </div>
  <div
    className="topTriangle container"
  >
    <div
      className="row"
    >
      <div
        className="col-md-6"
      >
        <div
          className="profileImage"
        >
          <Image
            source="/image/Mustafa.jpg"
            style={
              Object {
                "borderRadius": 150,
                "height": 300,
                "width": 300,
              }
            }
          />
          <div
            className="profileName"
          >
            
          </div>
        </div>
      </div>
      <div
        className="col-md-6"
      >
        <div
          className="bioText"
        >
          
        </div>
        <div
          className="bioText"
        >
          
        </div>
        <div
          className="bioText"
        >
          
        </div>
      </div>
      <div
        className="row"
      >
        <div
          className="buttondiv"
        >
          <button
            className="btn btn-primary button"
            onClick={[Function]}
            style={
              Object {
                "cursor": "pointer",
              }
            }
            testID="leftCVbutton"
            type="submit"
          >
            See CV
          </button>
          <button
            className="btn btn-primary button"
            onClick={[Function]}
            style={
              Object {
                "cursor": "pointer",
              }
            }
            testID="leftProjectbutton"
            type="submit"
          >
            See Personal Projects
          </button>
        </div>
      </div>
    </div>
  </div>
  <div
    className="bottomTriangle bgSecondary container"
  >
    <div
      className="row"
    >
      <div
        className="col-md-6"
      >
        <div
          className="profileImage"
        >
          <Image
            source="/image/Xu.jpg"
            style={
              Object {
                "borderRadius": 150,
                "height": 300,
                "width": 300,
              }
            }
          />
          <div
            className="profileName"
          >
            
          </div>
        </div>
      </div>
      <div
        className="col-md-6"
      >
        <div
          className="bioText"
        >
          
        </div>
        <div
          className="bioText"
        >
          
        </div>
        <div
          className="bioText"
        >
          
        </div>
      </div>
      <div
        className="row"
      >
        <div
          className="buttondiv"
        >
          <button
            className="btn btn-primary button"
            onClick={[Function]}
            style={
              Object {
                "cursor": "pointer",
              }
            }
            testID="rightCVbutton"
            type="submit"
          >
            See CV
          </button>
          <button
            className="btn btn-primary button"
            onClick={[Function]}
            style={
              Object {
                "cursor": "pointer",
              }
            }
            testID="rightProjectbutton"
            type="submit"
          >
            See Personal Projects
          </button>
        </div>
      </div>
    </div>
  </div>
  <div
    className="topTriangle container"
  >
    <div
      className="row"
    >
      <div
        className="col-md-6"
      >
        <div
          className="profileImage"
        >
          <Image
            source="/image/Mengyan.jpg"
            style={
              Object {
                "borderRadius": 150,
                "height": 300,
                "width": 300,
              }
            }
          />
          <div
            className="profileName"
          >
            
          </div>
        </div>
      </div>
      <div
        className="col-md-6"
      >
        <div
          className="bioText"
        >
          
        </div>
        <div
          className="bioText"
        >
          
        </div>
        <div
          className="bioText"
        >
          
        </div>
      </div>
      <div
        className="row"
      >
        <div
          className="buttondiv"
        >
          <button
            className="btn btn-primary button"
            onClick={[Function]}
            style={
              Object {
                "cursor": "pointer",
              }
            }
            testID="leftCVbutton"
            type="submit"
          >
            See CV
          </button>
          <button
            className="btn btn-primary button"
            onClick={[Function]}
            style={
              Object {
                "cursor": "pointer",
              }
            }
            testID="leftProjectbutton"
            type="submit"
          >
            See Personal Projects
          </button>
        </div>
      </div>
    </div>
  </div>
  <div
    className="getInTouchHeader"
  >
    Contact Us
  </div>
  <div>
    <div
      className="contact"
    >
      <div
        className="bioTextStable"
      >
        If you are interested in any of the projects or information you see on this page, please contact us below, or email us directly through one of the email addresses listed in our profiles. We look forward to hearing from you!  
        <p />
         
      </div>
      <form
        action="/email"
        id="contact-form"
        method="POST"
        onSubmit={[Function]}
      >
        <div>
          <div
            className="bioTextContact"
          >
            <label
              htmlFor="name"
            >
              Name
            </label>
          </div>
          <input
            className="form-control contactForm"
            onChange={[Function]}
            placeholder="Enter Name"
            type="text"
            value=""
          />
        </div>
        <div>
          <div
            className="bioTextContact"
          >
            <label
              htmlFor="exampleInputEmail1"
            >
              Email address
            </label>
          </div>
          <input
            aria-describedby="emailHelp"
            className="contactForm"
            onChange={[Function]}
            placeholder="Enter Email"
            type="email"
            value=""
          />
        </div>
        <div>
          <div
            className="bioTextContact"
          >
            <label
              htmlFor="message"
            >
              Message
            </label>
          </div>
          <div>
            <textarea
              className="messageForm"
              onChange={[Function]}
              placeholder="Enter Message"
              rows="5"
              value=""
            />
          </div>
        </div>
        <button
          className="btn btn-primary buttonsubmit"
          type="submit"
        >
          Submit
        </button>
      </form>
    </div>
  </div>
  <div
    className="footer"
  >
    <p>
      last edited: 21/10/2020
    </p>
  </div>
</div>
`;
